Can not use int in args input

I'm trying to use `int` as an input type but when I run following code it panics:

```
panic: bad method echo on type schemabuilder.mutation: attempted to parse  as arguments struct, but failed: bad arg type int: should be struct, scalar, pointer, or a slice

goroutine 1 [running]:
github.com/samsarahq/thunder/graphql/schemabuilder.(*Schema).MustBuild(...)
	/home/o/go/src/github.com/samsarahq/thunder/graphql/schemabuilder/schema.go:281
main.(*server).schema(0xc0001d1f78, 0x0)
	/home/o/go/src/github.com/sijad/testthunder/main.go:28 +0x86
main.main()
	/home/o/go/src/github.com/sijad/testthunder/main.go:34 +0x2f
exit status 2
```


### MRE

```go
package main

import (
	"net/http"

	"github.com/samsarahq/thunder/graphql"
	"github.com/samsarahq/thunder/graphql/schemabuilder"
)

// server is our graphql server.
type server struct{}

// registerMutation registers the root mutation type.
func (s *server) registerMutation(schema *schemabuilder.Schema) {
	obj := schema.Mutation()
	obj.FieldFunc("echo", func(args struct {
		Message    string
		MyIntValue int
	}) string {
		return args.Message
	})
}

// schema builds the graphql schema.
func (s *server) schema() *graphql.Schema {
	builder := schemabuilder.NewSchema()
	s.registerMutation(builder)
	return builder.MustBuild()
}

func main() {
	server := &server{}

	schema := server.schema()

	// Expose schema and graphiql.
	http.Handle("/graphql", graphql.Handler(schema))
	http.ListenAndServe(":3030", nil)
}
```

when I remove `MyIntValue int`, it works fine

--- a/graphql/schemabuilder/input.go
+++ b/graphql/schemabuilder/input.go
@@ -426,6 +426,16 @@ var scalarArgParsers = map[reflect.Type]*argParser{
 			return nil
 		},
 	},
+	reflect.TypeOf(int(0)): {
+		FromJSON: func(value interface{}, dest reflect.Value) error {
+			asFloat, ok := value.(float64)
+			if !ok {
+				return errors.New("not a number")
+			}
+			dest.Set(reflect.ValueOf(int(asFloat)).Convert(dest.Type()))
+			return nil
+		},
+	},
 	reflect.TypeOf(uint64(0)): {
 		FromJSON: func(value interface{}, dest reflect.Value) error {
 			asFloat, ok := value.(float64)
@@ -463,6 +473,16 @@ var scalarArgParsers = map[reflect.Type]*argParser{
 				return errors.New("not a number")
 			}
 			dest.Set(reflect.ValueOf(uint8(asFloat)).Convert(dest.Type()))
+			return nil
+		},
+	},
+	reflect.TypeOf(uint(0)): {
+		FromJSON: func(value interface{}, dest reflect.Value) error {
+			asFloat, ok := value.(float64)
+			if !ok {
+				return errors.New("not a number")
+			}
+			dest.Set(reflect.ValueOf(uint(asFloat)).Convert(dest.Type()))
 			return nil
 		},
 	},

--- a/graphql/schemabuilder/reflect_test.go
+++ b/graphql/schemabuilder/reflect_test.go
@@ -437,6 +437,8 @@ type kitchenSinkArgs struct {
 	Hello               int64
 	Hello32             int32
 	Hello16             int16
+	HelloInt            int
+	HelloUint           uint
 	HelloFloat32        float32
 	HelloFloat64        float64
 	FooBar              string
@@ -502,6 +504,8 @@ func TestArgParser(t *testing.T) {
 			"hello": 20,
 			"hello32": 20,
 			"hello16": 20,
+			"helloInt": 22,
+			"helloUint": 22,
 			"helloFloat32": 42.0,
 			"helloFloat64": 42.0,
 			"fooBar": "foo!",
@@ -517,6 +521,8 @@ func TestArgParser(t *testing.T) {
 		Hello:           20,
 		Hello32:         20,
 		Hello16:         20,
+		HelloInt:        22,
+		HelloUint:       22,
 		HelloFloat32:    42.0,
 		HelloFloat64:    42.0,
 		FooBar:          "foo!",
@@ -546,6 +552,8 @@ func TestArgParser(t *testing.T) {
 			"hello": 40,
 			"hello32": 40,
 			"hello16": 40,
+			"helloInt": 22,
+			"helloUint": 22,
 			"helloFloat64": 40.0,
 			"helloFloat32": 40.0,
 			"fooBar": "bar!",
@@ -569,6 +577,8 @@ func TestArgParser(t *testing.T) {
 		Hello:               40,
 		Hello32:             40,
 		Hello16:             40,
+		HelloInt:            22,
+		HelloUint:           22,
 		HelloFloat64:        40.0,
 		HelloFloat32:        40.0,
 		FooBar:              "bar!",

